packages/browser/src/index.test.ts

```diff
@@ -144,4 +159,81 @@ describe('LogtoClient', () => {
       );
     });
   });
+
+  describe('getAccessToken', () => {
+    test('should throw if idToken is empty', async () => {
+      localStorage.removeItem(idTokenStorageKey);
+      localStorage.setItem(refreshTokenStorageKey, 'refresh_token_value');
+      const logtoClient = new LogtoClient({ endpoint, clientId }, requester);
+
+      await expect(logtoClient.getAccessToken()).rejects.toThrow();
```

Comment:
better use `toMatchError` to match specific Error `new LogtoClientError('not_authenticated')`

Reply:
ok